src/components/CartoonCards.jsx

```diff
@@ -0,0 +1,26 @@
+import FavoriteCartoons from './FavoriteCartoons'
+
+export default function CartoonCards ({ cartoonsCard }){
+    return(
+        <>
+            <div>
+                { cartoonsCard.name }
```

Comment:
i don't think you need this

Reply:
Yes, I initially included it to test its functionality and unintentionally forgot to remove it. I have now removed it.
Thank you!